scripts/test-extensions-unit.sh

```diff
@@ -21,49 +22,62 @@ if [[ "$ADS_TEST_GREP" == "" ]]; then
 	export ADS_TEST_INVERT_GREP=1
 fi
 
+# Figure out which Electron to use for running tests
+if [ -z "$INTEGRATION_TEST_ELECTRON_PATH" ]
+then
+	# Run out of sources: no need to compile as code.sh takes care of it
+	INTEGRATION_TEST_ELECTRON_PATH="./scripts/code.sh"
+
+	echo "Running integration tests out of sources."
+else
+	# Run from a built: need to compile all test extensions
+
+	echo "Running integration tests with '$INTEGRATION_TEST_ELECTRON_PATH' as build."
+fi
+
 cd $ROOT
 echo $VSCODEUSERDATADIR
 echo $VSCODEEXTDIR
 
 echo ***************************************************
 echo *** starting admin tool extension windows tests ***
 echo ***************************************************
-./scripts/code.sh --nogpu --extensionDevelopmentPath=$ROOT/extensions/admin-tool-ext-win --extensionTestsPath=$ROOT/extensions/admin-tool-ext-win/out/test --user-data-dir=$VSCODEUSERDATADIR --extensions-dir=$VSCODEEXTDIR
+"$INTEGRATION_TEST_ELECTRON_PATH" $LINUX_NO_SANDBOX --nogpu --extensionDevelopmentPath=$ROOT/extensions/admin-tool-ext-win --extensionTestsPath=$ROOT/extensions/admin-tool-ext-win/out/test --user-data-dir=$VSCODEUSERDATADIR --extensions-dir=$VSCODEEXTDIR
 
```

Comment:
Not doing this for the batch file version as well?

Reply:
It wasn't necessary for this change, but I can do that.